docs/docs-content/clusters/pcg/deploy-pcg/vmware.md

```diff
@@ -0,0 +1,550 @@
+    | **Registry Username**                                    | The username for the custom registry.                                                                                                                                                                                                                              |
+    | **Password**                                             | The password for the custom registry.                                                                                                                                                                                                                              |
+
+6.  The next set of prompts is for configuring connection details for the vSphere environment. The CLI will use this
+    information to establish a network connection to the vSphere environment and query the vSphere API to retrieve
+    information about the environment. The information retrieved is used in the next step for you to select target
+    resources.
+
+    | **Parameter**                                            | **Description**                                                                                                                                                                                                                                                    |
+    | -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+    | **vSphere Endpoint**                                     | The vSphere endpoint. You can specify a full qualified domain name (FQDN) or an IP address. Make sure you specify the endpoint without the HTTP scheme `https://` or `http://`. Example: `vcenter.mycompany.com`                                                   |
+    | **vSphere Username**                                     | The vSphere account username.                                                                                                                                                                                                                                      |
+    | **vSphere Password**                                     | The vSphere account password.                                                                                                                                                                                                                                      |
+    | **Allow Insecure Connection (Bypass x509 Verification)** | Enabling this option bypasses x509 CA verification. Enter `n` if using a custom registry with self-signed SSL certificates. Otherwise, enter `y`. If you enter `y`, you will receive a follow up prompt asking you to provide the file path to the CA certificate. |
```

Comment:
```suggestion
    | **Allow Insecure Connection (Bypass x509 Verification)** | Enabling this option bypasses x509 CA verification. Enter `n` if using a custom registry with self-signed SSL certificates. Otherwise, enter `y`. If you enter `y`, you will receive a follow-up prompt asking you to provide the file path to the CA certificate. |
```